dice/dice_help.go

```diff
@@ -646,44 +646,50 @@ func (m *HelpManager) GetContent(item *HelpTextItem, depth int) string {
 		return "{递归层数过多，不予显示}"
 	}
 	txt := item.Content
-	finalTxt := ""
 	re := regexp.MustCompile(`\{[^}\n]+\}`)
 	matched := re.FindAllStringSubmatchIndex(txt, -1)
+	if len(matched) == 0 {
+		return txt
+	}
 
+	replaced := strings.Builder{}
+	handledRight := 0
```

Comment:
```suggestion
	lastHandledRight := 0
```

这俩变量名读的时候反应了半天，或许可以改改？别的都OK